vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1640,18 +1640,11 @@ class TimelineFragment @Inject constructor(
         }
     }
 
-    private fun renderSubTitle(typingMessage: String?, topic: String) {
-        // TODO Temporary place to put typing data
-        val subtitle = typingMessage?.takeIf { it.isNotBlank() } ?: topic
+    private fun renderSubTitle(topic: String) {
         views.includeRoomToolbar.roomToolbarSubtitleView.apply {
-            setTextOrHide(subtitle)
-            if (typingMessage.isNullOrBlank()) {
-                setTextColor(colorProvider.getColorFromAttribute(R.attr.vctr_content_secondary))
-                setTypeface(null, Typeface.NORMAL)
-            } else {
-                setTextColor(colorProvider.getColorFromAttribute(R.attr.colorPrimary))
-                setTypeface(null, Typeface.BOLD)
-            }
+            setTextOrHide(topic)
+            setTextColor(colorProvider.getColorFromAttribute(R.attr.colorPrimary))
```

Comment:
I think we want to keep `vctr_content_secondary` here. Or better declare it in the XML since it will not change anymore.

Reply:
Done.
Thanks